src/endpoints/comparisons/ComparisonsEndpoint.ts

```diff
@@ -3,7 +3,7 @@ import Urls from '../urls';
 import { ComparisonData, ComparisonsResult, Side, Stream } from './types';
 import AuthenticatedNeedleClient from '../../utilities/AuthenticatedNeedleClient';
 import Comparison from './Comparison';
-import allowedFileTypes from './consts';
+import AllowedFileTypes from './consts';
```

Comment:
I would usually recommend that constants be written like `ALLOWED_FILE_TYPES`, as TitleCase should be reserved for classes

Also I'm not sure there is any value in having the name of the file we're importing this from shortened (as it can hurt searchability).

e.g.
```
import ALLOWED_FILE_TYPESfrom './constants';
```